symplyphysics/laws/quantities/quantity_is_areal_density_times_area.py

```diff
@@ -0,0 +1,61 @@
+"""
+Quantity is areal density times volume
+======================================
+
+An extensive quantity of interest can be obtained by multiplying the corresponding *areal
+density* by area.
+"""
+
+from sympy import Eq, Symbol as SymSymbol
+from symplyphysics import (
+    units,
+    Quantity,
+    SymbolNew,
+    validate_input,
+)
+
+extensive_quantity = SymSymbol("extensive_quantity")
+"""
+Extensive quantity.
+
+Symbol:
+    :code:`X`
+"""
+
+areal_density = SymSymbol("areal_density")
+r"""
+Intensive area-specific density.
+
+Symbol:
+    :code:`sigma_X`
+
+Latex:
+    :math:`\sigma_X`
+"""
+
+area = SymbolNew("A", units.area)
+"""
+Area.
+"""
+
+law = Eq(extensive_quantity, areal_density * area)
+r"""
+:code:`X = sigma_X * A`
+
+Latex:
+    .. math::
+        X = \sigma_X A
+"""
+
+@validate_input(area_=area)
```

Comment:
In similar laws we validated that output dimension corresponds to input.

Reply:
I now think this is not needed. Multiplying anything by area would always result in a quantity whose dimension is equal to multiplying that dimension by the dimension of area. Perhaps if we included a dimension to check for as a parameter, this would be useful to have. But so far this check does nothing.

Reply:
It's true. But we also can show the output dimension this way. For example, `validate_output` never triggers, if formula is correct. But it shows the resulting dimension to the user.

Reply:
Hm, I see what you mean.